File: Java/ReverseString.java
public class StringReversal {
    public static String reverseString(String input) {
        StringBuilder reversed = new StringBuilder(input);
        reversed.reverse();
        return reversed.toString();
    }

    public static void main(String[] args) {
        String inputStr = "Hello, World!";
        String reversedStr = reverseString(inputStr);
        System.out.println("Reversed: " + reversedStr);
    }
}

File: Rust/ReverseString.rs
fn reverse_string(input: &str) -> String {
    let reversed: String = input.chars().rev().collect();
    return reversed;
}

fn main() {
    let input_str = "Hello, World!";
    let reversed_str = reverse_string(input_str);
    println!("Reversed: {}", reversed_str);
}